src/Standards/Generic/Sniffs/WhiteSpace/LanguageConstructSpacingSniff.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * Ensures all language constructs contain a single space between themselves and their content.
+ *
+ * @author    Greg Sherwood <gsherwood@squiz.net>
+ * @copyright 2006-2015 Squiz Pty Ltd (ABN 77 084 670 600)
+ * @license   https://github.com/squizlabs/PHP_CodeSniffer/blob/master/licence.txt BSD Licence
+ */
+
+namespace PHP_CodeSniffer\Standards\Generic\Sniffs\WhiteSpace;
+
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+
+class LanguageConstructSpacingSniff implements Sniff
+{
+
+
+    /**
+     * Returns an array of tokens this test wants to listen for.
+     *
+     * @return array
+     */
+    public function register()
+    {
+        return array(
+                T_ECHO,
+                T_PRINT,
+                T_RETURN,
+                T_INCLUDE,
+                T_INCLUDE_ONCE,
+                T_REQUIRE,
+                T_REQUIRE_ONCE,
+                T_NEW,
+                T_YIELD,
```

Comment:
hello @jrfnl

As I suspected this sniff is not working as expected with two word tokens (check the [travis](https://travis-ci.org/squizlabs/PHP_CodeSniffer/jobs/256926241))

Do you have any suggestion about this?

My suggestion would be:

- Remove these test cases but add comments on the sniff and inside test and fix this on a different PR.
- Remove the T_YIELD_FROM add comments on the sniff that T_YIELD_FROM needs to be added and solve this on a different PR.

@gsherwood  what do you think?